LLM/ocr_functions.py

```diff
@@ -0,0 +1,191 @@
+import base64
+import json
+import pandas as pd
+from PIL import Image, ExifTags
+from openai import OpenAI
+from msfocr.data import data_upload_DHIS2 as dhis2
+import configparser
+
+
```

Comment:
This isn't necessary for the code to work, but I like to declare my module constants all together at the top of the module, even if they aren't set in the code until later. This makes it easier to read the code and understand what's going on and what variables are important. 

```suggestion
OPENAI_API_KEY = None

```